db/seeds.rb

```diff
@@ -126,7 +131,12 @@ def create_users
                    last_name: last_name,
                    username: "#{first_name.downcase}_user",
                    password: "password",
-                   role: 0)
+                   role: 0,
+                   email_address: "#{first_name.downcase}_user@gmail.com",
+                   street_address: "123 Maple Drive",
+                   city: "Denver",
+                   state: "CO",
+                   zipcode: 80231)
```

Comment:
I don't think we need to do this with a zipcode, it doesn't make any sense and makes it more confusing to read.